scripts/mkcloudcloud/heat-setup.sh

```diff
@@ -6,7 +6,17 @@ if [ -z "$namespace" -o -z "$openstacksshkey" ] ; then
 fi
 
 stackname=mkcc-$namespace
-openstack stack create --wait -t setup.yaml --parameter key_name=$openstacksshkey --parameter namespace=$namespace $stackname
+if openstack stack show $stackname >/dev/null; then
+  echo "Stack $stackname already exists, deleting..."
+  until openstack stack delete -y --wait $stackname; do
+    sleep 1
+  done
+fi
+
+until openstack stack create --wait -t setup.yaml --parameter key_name=$openstacksshkey --parameter namespace=$namespace $stackname; do
+  sleep 1
```

Comment:
Why do we need to retry the stack create?

Reply:
During my tests I did re-deployed this stack several times. That's why there was a `stack delete` command. After a `stack delete` you cannot re-create the stack straight away. That's why there's a retry loop here.

Reply:
For now I'll leave the retry out of the script.